data/meson.build

```diff
@@ -94,12 +94,15 @@ if build_daemon
       dynamic_options += 'RestrictAddressFamilies=AF_NETLINK AF_UNIX'
     endif
 
-    # flashrom requires raw-io
-    system_call_filter = ['~@clock', '@cpu-emulation', '@debug', '@memlock', '@module', '@mount', '@obsolete', '@pkey', '@reboot', '@resources', '@swap']
-    if not allow_flashrom
-      system_call_filter += ['@raw-io']
+    # the order of SystemCallFilter is important as the first encountered will take precedence and
+    # will dictate the default action
+    syscall_allow = ['@system-service']
+    syscall_deny = ['@resources']
+    if allow_flashrom
+      syscall_allow += ['@raw-io']
     endif
-    dynamic_options += ['SystemCallFilter=' + ' '.join(system_call_filter)]
+    dynamic_options += ['SystemCallFilter=' + ' '.join(syscall_allow)]
+    dynamic_options += ['SystemCallFilter=~' + ' '.join(syscall_deny)] # tilde is not a typo
```

Comment:
does the tilde need to go on each item or once for the line?

Reply:
The very first character only -- but please check my sanity here: https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html

Reply:
Ah yup, that looks correct.